package.json

```diff
@@ -28,6 +29,7 @@
     "formik": "2.2.9",
     "immer": "9.0.6",
     "in-browser-download": "2.0.0",
+    "jotai": "^2.10.1",
```

Comment:
Notifications application is using Redux. I don't think it's reasonable to add Jotai while adding a the drawer to it.